spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/FilterQuery.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spark.bigquery.pushdowns
+
+import com.google.cloud.spark.bigquery.pushdowns.SparkBigQueryPushdownUtil.makeStatement
+import org.apache.spark.sql.catalyst.expressions.Expression
+
+/** Query for filter operations.
+ *
+ * @constructor
+ * @param conditions The filter condition.
+ * @param child      The child node.
+ * @param alias      Query alias.
+ */
+case class FilterQuery(
+    expressionConverter: SparkExpressionConverter,
+    conditions: Seq[Expression],
+    child: BigQuerySQLQuery,
+    alias: String)
+  extends BigQuerySQLQuery(
+    expressionConverter,
+    alias,
+    children = Seq(child)) {
+
+  /** Builds the WHERE statement of the filter query */
+  override val suffixStatement: BigQuerySQLStatement =
+    ConstantString("WHERE") + makeStatement(
+      conditions.map(expressionToStatement),
+      "AND"
```

Comment:
Surround each condition with parenthesis so it would be `WHERE (...) AND (...)`. If `expressionToStatement` takes care of it then ignore

Reply:
yup. that is exactly what happens. We can see it in the unit test in FilterQuerySuite https://github.com/GoogleCloudDataproc/spark-bigquery-connector/pull/600/files#diff-1ba967ace855ade5c727be80a4c4a5702644328d349253f1306d1645ba9cef2cR35

```
test("suffixStatement") {
    assert(filterQuery.suffixStatement.toString == "WHERE ( SUBQUERY_0.SCHOOLID >= 50 ) AND ( SUBQUERY_0.SCHOOLID <= 100 )")
  }
```